docker-compose.yaml

```diff
@@ -0,0 +1,7 @@
+services:
+
+  nginx-angular:
+    container_name: nginx-angular
+    image: ghcr.io/lepavuki/angular-react-devops-case:pr-3
```

Comment:
Pls update your workflow to produce nice tags for docker images, i.e. should be at least `latest` one

Reply:
Added "latest" tag handling